### ISSUE TITLE
Nested Constellations

### ISSUE THREAD

**Issue by AntoineCharton** [AntoineCharton/Constellation#13]
- Add a constellation node
- It will be able to store an other constellation

**Comment by eggeuk** [comment]
Why long therm? Is it difficult?

**Comment by AntoineCharton** [comment]
Yes, I'm very close to have it, but it's in deed very difficult. There is a lot of possible issues it can cause.
